source/php-libraries.txt

```diff
@@ -12,133 +12,83 @@ PHP Libraries, Frameworks, and Tools
    :depth: 2
    :class: singlecol
 
-Libraries for the ``mongodb`` Extension
----------------------------------------
+Standalone Libraries
+--------------------
 
-Stand-alone Libraries
-~~~~~~~~~~~~~~~~~~~~~
+- `Doctrine MongoDB ODM <https://github.com/doctrine/mongodb-odm>`__ is a
+  library that provides object mapping functionality for MongoDB.
+  You can use this library standalone or use one of the following framework
```

Comment:
S: I know exactly what this means, but I'e never seen "standalone" used as an adverb

```suggestion
  You can use this library on its own or use one of the following framework
```

(or something similar)

Reply:
Applies to all occurrences of "standalone" as adverb